docs/content/concepts/identifier-use/_index.md

```diff
@@ -0,0 +1,212 @@
+
+#### Human-Oriented
+
+A *`human-oriented`* identifier incorporates semantics that support readability and processing by humans. OSCAL implements *`human-oriented`* identifiers as Token data types, which are non-colonized names (add link to data type). For example, control identifiers in a catalog may use a nomenclature that is familiar to the intended audience, allowing them to quickly determine what security control is being referred to, simply by its identifier value.  
+
+The [OSCAL Reference Index](/reference/latest/complete/xml-index/#/@id) provides a comprehensive listing of the *`human-oriented`* IDs in OSCAL.  References to these IDs are typically named according to the referenced object type (e.g., control) followed by “-id”, as seen here in the [Reference Index](/reference/latest/complete/xml-index/#/@control-id).
+
+### **Uniqueness** 
+OSCAL identifier uniqueness is categorized as *`locally-unique`* or *`globally-unique`*.  As implied by the category name, *`locally-unique`* identifiers must be unique within the current document, whereas *`globally-unique`* identifiers are guaranteed to be unique across all other identifiers.  OSCAL’s *`machine-oriented`* UUID identifiers are always *`globally-unique`*.  *`Human-oriented`* identifiers must be defined and managed organizationally and are more susceptible to identifier duplication or collisions. Thus, *`human-oriented`* identifiers are less likely or cannot be guaranteed to be *`globally-unique`*.  
+
+### **Scope** 
+Identifiers are declared so that data items in OSCAL instances can be referenced accurately and consistently.  In the most basic instance, references are made to data items in the same OSCAL instance, but because OSCAL supports instance importing, there are scenarios where data items in an external / imported OSCAL instance need to be referenced as well.  Identifiers that are only intended for use within the same OSCAL instance are categorized as *`instance`* scope, but identifiers that can be referenced (from other OSCAL instances) are considered *`cross-instance`* scoped. While *`human-oriented`* identifiers might not be globally unique, they may be *`cross-instance`* scoped.  In these cases, the appropriate import attribute (e.g. import, import-component-definitions, import-profile, import-ssp, and import-ap) should be used (similar to a namespacing) to deconflict identifiers with the same values in the associated OSCAL instances.  This would allow for the same control IDs to be used and referenced in a profile and its imported catalog(s) for example.  The tables below provides a summary of OSCAL unique identifier scoping.
+
+#### **Catalog Identifiers**
+Identifiers defined in a catalog may be referenced locally or from an importing profile.
+
+|**Defining Model**|**Identifier**|**Catalog**|**Profile**|**Component**|**SSP**|**AP**|**AR**|**POA&M**|
```

Comment:
General feedback on the headers, we might want to consider saying, like in this catalog example:

```suggestion
|**Defining Model**|**Identifier**|**Self**|**Profile**|**Component**|**SSP**|**AP**|**AR**|**POA&M**|
```

or

```suggestion
|**Defining Model**|**Identifier**|**Catalog (Self)**|**Profile**|**Component**|**SSP**|**AP**|**AR**|**POA&M**|
```

Either that, or we mean "this document instance, but other catalogs can cross-reference each other!" but I am not sure 1) what is possible for each of the document instances and 2) if we want to muddy the waters and bring that up. Does that make sense?

Reply:
This is obsolete now.